assets/js/DataTables-1.10.16/datatables.css

```diff
@@ -822,6 +822,9 @@ a.dt-button.processing:after {
 }
 
 
+table thead tr th input {
```

Comment:
Comment that this is datatables customization, or otherwise rename the file datatables_custom.css and use it that way somehow.  I think this is the proper way to go about modifying a third party asset.  Anyone else have an opinion on this?